### ISSUE TITLE
"Control 55 in window 10025 has been asked to focus, but it can't"

### ISSUE THREAD

**Issue by Astaoth** [webmeister/plugin.video.youtube.channels#2]
Hi !

I have Kodi running on a raspi 4 with Raspbian. When trying to launch this plugin, I have this errors : 
```
2020-02-25 19:24:47.702 T:2973232224   ERROR: Control 55 in window 10025 has been asked to focus, but it can't
2020-02-25 19:24:49.260 T:2087706832  NOTICE: script.module.youtube.dl: youtube_dl core version: 2019.09.12.1
2020-02-25 19:24:49.545 T:2689589456   ERROR: GetDirectory - Error getting plugin://plugin.video.youtube.channels/
2020-02-25 19:24:49.556 T:2973232224   ERROR: CGUIMediaWindow::GetDirectory(plugin://plugin.video.youtube.channels/) failed
```
I don't understand what means the first error, but since Kodi on a raspi takes control on the graphical stack, I don't know if it can have windows like the one in this error.
@webmeister Do you use this plugin on a Kodi installed on a raspi or a standard computer ?

**Comment by webmeister** [comment]
I see a similar message on x86_64, but everything seems to work nonetheless. The actual problem in this case is the GetDirectory error, which has been fixed together with #1.

**Comment by Astaoth** [comment]
Indeed, after fixing the other bug, the plugin works fine. Thank you ! :D

Edit : well after having played a little with it, I have some improvement ideas, I'll put them in new issues. I think they could be usefull for a better popularity, since the other youtube plugin needs API keys. But for me, once I know how it works, it's fine :)
